### PR TITLE
Use more condensed output

### PR DESCRIPTION
**Pull request by mhahn**
This also adds a `--replacements-only` flag which will only prompt for approval when the resource requires replacement

### CODE REVIEW

**Comment by mhahn** [comment]
Default output:
```
[2016-08-02T16:12:09] mh-empire-dev-empireDaemon changes:
Replacements:
- Modify TaskDefinition (AWS::ECS::TaskDefinition)
Changes:
- Modify Service (AWS::ECS::Service)
Execute the above changes? [y/n/v] v
```

Output with `--replacements-only`:
```
[2016-08-02T16:06:46] mh-empire-dev-empireDaemon replacements:
- Modify TaskDefinition (AWS::ECS::TaskDefinition)
Execute the above changes? [y/n/v] n
```

File: stacker/providers/aws/interactive.py [hunk 1]
```diff
@@ -24,9 +24,73 @@ def get_change_set_name():
...
 
 
+def requires_replacement(changeset):
+    """Return the changes within the changeset that require replacement.
+
+    Args:
+        changeset (List): List of changes
+
+    Returns:
+        List: A list of changes that require replacement, if any.
```
Comment (phobologic) [inline stacker/providers/aws/interactive.py]: nit: `List` -> `list`

File: stacker/providers/aws/interactive.py [hunk 2]
```diff
@@ -24,9 +24,73 @@ def get_change_set_name():
...
+
+    """
+    return [r for r in changeset if r["ResourceChange"]["Replacement"] ==
+            'True']
+
+
+def ask_for_approval(full_changeset=None, include_verbose=False):
```
Comment (phobologic) [inline stacker/providers/aws/interactive.py]: docstring

File: stacker/providers/aws/interactive.py [hunk 3]
```diff
@@ -24,9 +24,73 @@ def get_change_set_name():
...
+        logger.info(
+            "Full changeset:\n%s",
+            yaml.safe_dump(full_changeset),
+        )
+        return ask_for_approval()
+    elif approve != "y":
+        raise exceptions.CancelExecution
+
+
+def output_summary(fqn, action, changeset, replacements_only=False):
```
Comment (phobologic) [inline stacker/providers/aws/interactive.py]: docstring

**Comment by phobologic** [comment]
Other than the nits/docstrings, this looks good! 👍

**Pull request merged**